### PR TITLE
Create service to send question to BE & fix issues with run app

### PR DESCRIPTION
**Pull request by MikolaLvivUA**
+ fix issues with run app
+ connect mongoDB
+ create mongoDB document scheme
+ create question service
+ create question controller

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -25,15 +26,24 @@ class App {
...
         this.mountRoutes();
+        this.setupDB();
 
         this.app.use(this.logErrors);
         this.app.use(this.clientErrorHandler);
         this.app.use(this.customErrorHandler);
     }
+    /*Mongo Connection*/
+    private setupDB(): void {
+        const mongoDB = 'mongodb://127.0.0.1:27017/QuestionDB';
```
Comment (VictorKmin) [inline src/app.ts]: Replace port and IP to constants

File: src/database/models/question.schema.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import * as mongoose from 'mongoose';
+import { config } from '../../configs';
```
Comment (VictorKmin) [inline src/database/models/question.schema.ts]: empty line

File: src/routes/question/question.router.ts [hunk 3]
```diff
@@ -1,5 +1,8 @@
-import {Router} from 'express'
+import { Router } from 'express';
+import { questionController } from '../../controllers/question';
```
Comment (VictorKmin) [inline src/routes/question/question.router.ts]: ../../controllers
Empty line

File: src/app.ts (hunk 1 above)
Comment (MikolaLvivUA) [reply src/app.ts]: fixed

File: src/database/models/question.schema.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/database/models/question.schema.ts]: fixed

File: src/routes/question/question.router.ts (hunk 3 above)
Comment (MikolaLvivUA) [reply src/routes/question/question.router.ts]: fixed

**Pull request merged**